tests/validation/tests/v3_api/test_app.py

```diff
@@ -1,20 +1,42 @@
 from .common import *  # NOQA
 import pytest
-
-project_detail = {"project": None, "namespace": None, "cluster": None,
-                  "project2": None, "namespace2": None, "cluster2": None}
-user_token = {"user_c1_p1_owner": {"user": None, "token": None},
-              "user_c1_p1_member": {"user": None, "token": None},
+from .test_rbac import create_user
+
+project_detail = {"cluster1": {"project1": rbac_data["project"],
+                               "namespace1": rbac_data["namespace"],
+                               "project2": None, "namespace2": None,
+                               "cluster": None},
+                  "cluster2": {"project1": None, "namespace1": None,
+                               "cluster": None}}
+user_token = {"user_c1_p1_owner": {
+                  "user": rbac_data["users"][PROJECT_OWNER]["user"],
+                  "token": rbac_data["users"][PROJECT_OWNER]["token"]},
+              "user_c1_p1_member": {
+                  "user": rbac_data["users"][PROJECT_MEMBER]["user"],
+                  "token": rbac_data["users"][PROJECT_MEMBER]["token"]},
               "user_c1_p2_owner": {"user": None, "token": None},
+              "user_c2_p1_owner": {"user": None, "token": None},
+              "user_c1_owner": {
+                  "user": rbac_data["users"][CLUSTER_OWNER]["user"],
+                  "token": rbac_data["users"][CLUSTER_OWNER]["token"]},
+              "user_c1_member": {
+                  "user": rbac_data["users"][CLUSTER_MEMBER]["user"],
+                  "token": rbac_data["users"][CLUSTER_MEMBER]["token"]},
+              "user_c2_owner": {"user": None, "token": None},
               "user_standard": {"user": None, "token": None}}
-
-CATALOG_URL = "https://git.rancher.io/charts"
-MYSQL_EXTERNALID_037 = "catalog://?catalog=library&template=mysql" \
-                       "&version=0.3.7"
-MYSQL_EXTERNALID_038 = "catalog://?catalog=library&template=mysql" \
-                       "&version=0.3.8"
-WORDPRESS_EXTID = "catalog://?catalog=library&template=wordpress" \
-                  "&version=1.0.5"
+CATALOG_NAME = random_test_name("test-catalog")
+PROJECT_CATALOG = random_test_name("test-pj")
+CLUSTER_CATALOG = random_test_name("test-cl")
+PROJECT_CATLG_EXTERNAL_ID = None
+CLUSTER_CATLG_EXTERNAL_ID = None
+CATALOG_URL = "https://github.com/sowmyav27/rancher-charts.git"
```

Comment:
Ultimately, I would like to see us have a repo for test-charts hosted in Rancher github, rather than using from a personal fork, but we can do this later if necessary.

Reply:
done